code/modules/client/verbs/suicide.dm

```diff
@@ -92,7 +92,16 @@
 									"[src] is aggressively grabbing [p_their()] own neck! It looks like [p_theyre()] trying to commit suicide.", \
 									"[src] is pulling [p_their()] eyes out of their sockets! It looks like [p_theyre()] trying to commit suicide.")
 		else if(a_intent == INTENT_HELP)
-			suicide_message = pick("[src] is hugging [p_them()]self to death! It looks like [p_theyre()] trying to commit suicide.", \
+			var/obj/item/organ/brain/userbrain = getorgan(/obj/item/organ/brain)
+			if(userbrain.damage >= 75)
```

Comment:
```suggestion
			if(userbrain?.damage >= 75)
```
As weird as it may be, people without brains can still potentially suicide. See: HARS. This means there is infact a potential case where a user will have no brain, getorgan will return null and then you'll runtime here.

This code suggestion prevent that.

Reply:
fear not timberpoes, I was able to fix the code by doing exactly what you said